src/app/services/authguard.service.ts

```diff
@@ -0,0 +1,29 @@
+import { TokenStorageService } from 'src/app/services/token-storage.service';
+import { Injectable } from '@angular/core';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/SimpleSurveyProject/SimpleSurvey-Frontend/pullRequest?prid=7430210)
```suggestion
import { Injectable } from "@angular/core";
```